### PR TITLE
feat: migrate webhook dispatches

### LINKED ISSUE #14: Migrate Webhook dispatches
- [ ] file structure -> based on path name, e.g /openapi/paths/actor-builds
- [ ] Fix operationId - make it camelCased - e.g GetListOfKeyValueStores
- [ ] File name of endpoints -> remove v2@ e.g. actor-tasks@{actorTaskId}@input.yaml

### PR DESCRIPTION
**Pull request by Jkuzz**
Resolves #14

### CODE REVIEW

**Review by HonzaTuron — APPROVED** [review]

File: openapi/components/schemas/Getcollectionresponse.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ type: object
 properties:
```
Comment (drobnikj) [inline openapi/components/schemas/Getcollectionresponse.yaml:5]: Let's do the filename proper CamelCased.

File: openapi/components/schemas/GetListOfWebhookDispatchesResponse.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+title: GetListOfWebhookDispatchesResponse
+required:
+  - data
+type: object
+properties:
+  data:
+    allOf:
```
Comment (drobnikj) [inline openapi/components/schemas/GetListOfWebhookDispatchesResponse.yaml:7]: Is allOf nessesary here?

**Review by drobnikj — CHANGES_REQUESTED** [review]

File: openapi/paths/webhooks/webhooks@{webhookId}@dispatches.yaml [hunk 3]
```diff
@@ -30,7 +30,7 @@ get:
         application/json:
           schema:
             allOf:
-              - $ref: ../components/schemas/Getcollectionresponse.yaml
```
Comment (tobice) [inline openapi/paths/webhooks/webhooks@{webhookId}@dispatches.yaml:33]: Can we reuse `GetListOfWebhookDispatchesResponse`?

File: openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml [hunk 4]
```diff
@@ -28,12 +28,10 @@ properties:
     type: string
     example: ACTOR.RUN.SUCCEEDED
   eventData:
-    allOf:
-      - $ref: ./eventData.yaml
-      - {}
+    $ref: ../eventData.yaml
```
Comment (HonzaTuron) [inline openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml:31]: How about renaming this as well?

File: openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml [hunk 5]
```diff
@@ -28,12 +28,10 @@ properties:
...
   calls:
     type: array
     items:
-      $ref: ./calls.yaml
+      $ref: ../calls.yaml
```
Comment (HonzaTuron) [inline openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml:35]: And this?

**Review by HonzaTuron — CHANGES_REQUESTED** [review]

**Review by tobice — APPROVED** [review]
Optimistically approving but IMHO we can get rid of one file there :smile:

File: openapi/paths/webhooks/webhooks@{webhookId}@dispatches.yaml (hunk 3 above)
Comment (Jkuzz) [reply openapi/paths/webhooks/webhooks@{webhookId}@dispatches.yaml:33]: Yes, reused 👍

File: openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml (hunk 4 above)
Comment (Jkuzz) [reply openapi/components/schemas/webhook-dispatches/WebhookDispatch.yaml]: Inlined these two, they were not used elsewhere

**Review by HonzaTuron — APPROVED** [review]

**Pull request merged**